src/components/FilterInput.svelte

```diff
@@ -1,5 +1,5 @@
 <script>
-  import throttle from "just-throttle";
+  const throttle = require("just-throttle");
```

Comment:
I think we should probably not make these changes, as the `import` syntax is more intuitive to people. I gather you're seeing some kind of error here? Can you post a trace and/or Google the results? I'm guessing that we probably just need to add a pre/post processing step somewhere in the build configuration.

Reply:
You are right. Currently I'm getting an error similar to this issue [https://github.com/facebook/jest/issues/6229](https://github.com/facebook/jest/issues/6229). 

- This Issue arises when an external library is used e.g. `marked` or `just-throttle` that is not transpiled to make ES6 imports work with nodejs. 

- `babel-jest` ignores node_modules altogether. I tried different plugins to make it work e.g.  `@babel/plugin-transform-modules-commonjs`. I also tried different solutions provided in the aforementioned issue, but still facing the same issue.

- Here is what I get when I try to use `marked`
![Screenshot 2020-11-18 at 10 27 21 PM](https://user-images.githubusercontent.com/19612755/99565357-57aee600-29ed-11eb-8af7-440e4d7b2f6f.png)

Reply:
Ah ok. Went down a bit of a rabbit hole with this. After looking at things a bit more, I think this has to do with the particulars of how these two libraries work as you describe. In the case of `just-throttle`, the solution is probably just to use [lodash's throttle function](https://lodash.com/docs/4.17.15#throttle) instead (this will require depending on lodash, which we want to do anyway IMO). In the case of marked, it's actually just a minor change we need to make in how it's imported. I'll suggest it.

Reply:
Oops! I spoke too soon. We actually *can't* use require, as it's not recognized by client-side JavaScript:

<img width="520" alt="image" src="https://user-images.githubusercontent.com/20569/99595182-4a303680-29c2-11eb-8bcc-d25913c73ef1.png">

Do you want to just go ahead and add lodash to the repo as part of this PR? Just run `npm install lodash` and you should be good to go.

Reply:
Yes, I will make the suggested changes in the next commit. Thanks.